images/manageiq-base/Dockerfile

```diff
@@ -45,46 +45,65 @@ LABEL name="manageiq-base" \
+
+RUN dnf -y --disableplugin=subscription-manager install http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-repos-8.1-1.1911.0.8.el8.${ARCH}.rpm http://mirror.centos.org/centos/8/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-8.1-1.1911.0.8.el8.noarch.rpm https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
     dnf -y --disableplugin=subscription-manager module enable nodejs:12 && \
     dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install \
-      chrony            \
-      cmake             \
-      cronie            \
-      file              \
-      gcc-c++           \
-      git               \
-      glibc-langpack-en \
-      hostname          \
-      http-parser       \
-      initscripts       \
-      iproute           \
-      libcurl-devel     \
-      libpq             \
-      libtool           \
-      libxml2-devel     \
-      libxslt-devel     \
-      logrotate         \
-      lvm2              \
-      make              \
-      net-tools         \
-      nmap-ncat         \
-      nodejs            \
-      openldap-clients  \
-      openscap-scanner  \
-      openssl-devel     \
-      patch             \
-      psmisc            \
-      postgresql-devel  \
-      redhat-rpm-config \
-      ruby-devel        \
-      sqlite-devel      \
-      wget              \
-      which             \
-      xz                \
-      &&                \
+      ansible                     \
+      ansible-runner              \
+      cmake                       \
+      cockpit-ws                  \
+      cockpit-ssh                 \
+      cyrus-sasl                  \
+      cyrus-sasl-plain            \
+      file                        \
+      freeipmi                    \
+      gcc-c++                     \
+      git                         \
+      glibc-langpack-en           \
+      hostname                    \
+      http-parser                 \
+      ipmitool                    \
+      iproute                     \
+      libcurl-devel               \
+      libpq                       \
+      libssh2-devel               \
+      libtool                     \
+      libxml2-devel               \
+      libxslt-devel               \
+      logrotate                   \
+      lvm2                        \
+      make                        \
+      net-snmp                    \
+      net-snmp-libs               \
+      net-snmp-utils              \
+      nmap-ncat                   \
+      nodejs                      \
+      openldap-clients            \
+      openscap-scanner            \
+      openssl-devel               \
+      OpenIPMI                    \
+      patch                       \
+      postfix                     \
+      postgresql-devel            \
+      procps-ng                   \
+      python3-devel               \
+      qpid-proton-c-devel         \
```

Comment:
If we're adding the `qpid-proton-c-devel` library, we should run bundle with `--with qpid_proton` option.